### PR TITLE
[RPL2-61] Scaffold publication content type mapping

### PR DESCRIPTION
**Pull request by waitingallday**
## Motivation and Context

**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-

## Changed

1.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist

- [ ] I've added relevant changes to the documentation.
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] My change requires a template update for create-ripple-app.
- [ ] I have added template update script for next release.

### CODE REVIEW

**Comment by waitingallday** [comment]
I decided to reduce the central mapping `index.js` down so that the mapping for Publication and PublicationPage can be used standalone, if that ever came up

**Comment by waitingallday** [comment]
Refactored all of the landing-page component stuff out into `ripple-tide-api` as well

File: packages/ripple-tide-api/src/services/utils/mapping/body.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default {
                   ['field_paragraph_accordion_name'],
                   ''
                 ),
-                content: getBodyFromField(content, 'field_paragraph_accordion_body')
+                content: getBodyFromField(content, 'field_paragraph_accordion_body').html
```
Comment (dylankelly) [inline packages/ripple-tide-api/src/services/utils/mapping/body.js:35]: If we end up deprecating the way RplMarkup works with client side vue components, we can make getBodyFromField just return the html

**Review by dylankelly — APPROVED** [review]
Nice one!

**Pull request closed without merging**